web/js/dynamicgraphview.js

```diff
@@ -0,0 +1,145 @@
+class DynamicGraphView extends GraphView {
```

Comment:
Later, the usefulness of this view should be discussed because most of the information this view provides are duplicated with timeline view.